tests/android/tests/test_android_discovery.py

```diff
@@ -134,8 +141,10 @@ def test_discovery_popular_subjects(self, set_capabilities, setup_logging):
         math_course = global_contents.wait_and_get_element(set_capabilities, android_elements.Discovery_third_popular_course)
         assert math_course.get_attribute('content-desc') == values.DISCOVERY_POPULAR_MATH_COURSE
         math_course.click()
+        sleep(10)
```

Comment:
Can we use wait or other function instead of waiting for certain seconds.

Reply:
@jawad-khan,  This is a webview page, and its loading time varies on multiple factors, 
adding a dynamic wait fails sometimes, so we have to wait for certain seconds to load it,